src/components/Post.tsx

```diff
@@ -91,14 +92,20 @@ function Post({
   useEffect(() => {
     // Fetch all the comments of the post from the API
     function fetchComments(authorId: string, postId: string) {
-      getAllComments(authorId.toString(), postId)
+      getAllComments(authorId, postId)
         .then((data) => setComments(data))
         .catch((err) => {
           setErrMsg(err.message);
         });
     }
 
+    function getLikesForPosts(authorId: string, postId: string){
```

Comment:
don't add extra functionalities like likes for posts. We are checking comments rn